examples/jenkins/pipeline/bluegreen-pipeline.yaml

```diff
@@ -59,7 +59,7 @@ objects:
             }
 
             stage("Go Live") {
-              sh "oc set -n ${project} route-backends ${appName} ${appName}-${tag}=100 ${appName}-${altTag}=0"
+              sh "oc set -n ${project} route-backends ${appName} ${appName}-${tag}=100 ${appName}-${altTag}=0 --loglevel=4"
```

Comment:
ah - the answer is yes (in `func debugAnyJenkinsFailure`).

[I tried replying via e-mail from my mobile to GitHub yesterday evening, and clearly it didn't work!]